qick_lib/qick/asm_v1.py

```diff
@@ -1491,6 +1460,11 @@ def __exit__(self, type, value, traceback):
         """
         pass
 
+class AcquireProgram(AcquireMixin, QickProgram):
```

Comment:
Will the init-dunder get called? I think the MRO will resolve to calling only the init-dunder of the first class

```python
class One():
    def __init__(self):
        print("One")
        
class Two():
    def __init__(self):
        print("Two")
        
class Both(One, Two):
    pass

Both() # prints 'One'
```

Reply:
Nevermind. I'm wrong